packages/react-menu/etc/react-menu.api.md

```diff
@@ -17,6 +17,15 @@ import { usePopperMouseTarget } from '@fluentui/react-positioning';
 // @public
 export const Menu: React_2.FC<MenuProps>;
 
+// @public (undocumented)
+export type MenuCheckedValueChangeData<TCheckedItems = string> = {
```

Comment:
I'm a TS noob, does this make it so that values must at least be strings ?

```suggestion
export type MenuCheckedValueChangeData<TCheckedItems extends string = string> = {
```

Reply:
This `<TCheckedItems = string>` defaults to string, and this `<TCheckedItems extends string = string>` default to string and limits values to at least be a string